.github/ISSUE_TEMPLATE/07-optimization.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: "Internal: proposed optimization"
+description: A proposal for a new query optimization. Contributor use only.
+labels: ["A-optimization", "C-feature", "needs-discussion"]
+body:
+  - type: textarea
+    attributes:
+      label: Elevator pitch
+      description: |
+        Each question should be answered comprehensively but using as few sentences as possible. This is the only mandatory section needed for the issue to be discussed in a refinement session.
+      value: |
+        #### What does the transformation do?
+        <!-- Give a representative SQL query. -->
+
+        #### Why should we add it?
+
+        #### When would it be good to have?
+
+        #### When would it be ineffectual?
+
+        #### When would be bad to have?
+
+        #### In the worst case, how would it degrade performance?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Customer impact
+      description: |
+          List real life instances where this transformation would help.
```

Comment:
This may be redundant given the questions under elevator pitch?

Reply:
As discussed offline, the confusion here comes from the similarity between the description of the "Customer impact" section and some of the question in the "Elevator Pitch".

The way I understand it, the "Customer impact" is more about general analysis, so talking about query classes and distribution of those classes some user workload makes sense. Maybe just rename the section to "Customer impact analysis" and refine what is respected in addition to the information already present in the "Elevator pitch".